src/main/kotlin/com/toasttab/pulseman/files/ZipManagement.kt

```diff
@@ -66,24 +67,32 @@ class ZipManagement(private val homeDirectory: String) {
 
     private fun savePath(file: File) = File(homeDirectory).toURI().relativize(file.toURI()).path
 
-    fun unzipProject(zippedFile: File): String? {
+    fun unzipProject(zippedFile: File, setUserFeedback: (String) -> Unit): String? {
         var projectJson: String? = null
         FileInputStream(zippedFile).use { fis ->
             BufferedInputStream(fis).use { bis ->
                 ZipInputStream(bis).use { zis ->
                     while (true) {
                         val nextEntry = zis.nextEntry ?: return projectJson
                         if (!nextEntry.isDirectory && nextEntry.name !in skipFiles) {
-                            if (nextEntry.name == projectTabsFileName) {
+                            val newFile = File("$homeDirectory${nextEntry.name}")
+                            val parentFile: File? = newFile.parentFile
+                            if (parentFile?.exists() == false && isValidFolderPrefix(parentFile.name)) {
+                                parentFile.mkdirs()
+                            }
```

Comment:
Need to create the tab folders now dynamically, we dont just have one folder anymore to store the jars